README.md

```diff
@@ -228,18 +228,20 @@ It will be passed as an argument to your function when it receives a request.
 Note that your function must use the `cloudevent`-style function signature:
 
 ```js
-exports.helloCloudEvents = (cloudevent) => {
+const functions = require('@google-cloud/functions-framework');
+
+functions.cloudEvent('helloCloudEvents', (cloudevent) => {
   console.log(cloudevent.specversion);
   console.log(cloudevent.type);
   console.log(cloudevent.source);
   console.log(cloudevent.subject);
   console.log(cloudevent.id);
   console.log(cloudevent.time);
   console.log(cloudevent.datacontenttype);
-};
+});
 ```
 
-To enable CloudEvents, set the signature type to `cloudevent`. By default, the HTTP signature will be used and automatic event unmarshalling will be disabled.
+To enable the CloudEvent functions, you must list the Functions Framework as a dependency in your `package.json`.
```

Comment:
Can we add the `package.json` snippet here?
```json
{
  "dependencies": {
    "@google-cloud/functions-framework": "some version we recommend",
    }
}
```

Since people can deploy functions without testing/having `npm` locally.

Reply:
sure